src/clusterfuzz/_internal/bot/fuzzers/engine_common.py

```diff
@@ -451,16 +451,12 @@ def get_all_issue_metadata(fuzz_target_path):
   return metadata
 
 
-def get_all_issue_metadata_for_testcase(testcase):
+def get_fuzz_target_issue_metadata(fuzz_target):
```

Comment:
We used to check whether the fuzz_target is None, how come we don't anymore?

Reply:
We do now, this was clearly a mistake. I thought it would be handled before calling this.
https://github.com/google/clusterfuzz/pull/3898

Reply:
Great, thanks!